client/src/components/RelatedProducts/contexts/FeatureContext.jsx

```diff
@@ -3,30 +3,30 @@ import React, {
   useState, useEffect, useContext, createContext,
 } from 'react';
 import axios from 'axios';
-import { ProductIDContext, useCurrentProductId } from '../../../contexts/ProductIDContext';
+import Console from '../../../Console';
+import { useCurrentProductId } from '../../../contexts/ProductIDContext';
 
 const FeatureContext = createContext();
```

Comment:
General note again - i wonder how much of this needed to live as a Context vs just passing things in as props and doing the computations in the individual components. Either way, it works so it's fine for now!